pom.xml

```diff
@@ -3,26 +3,98 @@
         <modelVersion>4.0.0</modelVersion>
         <groupId>pro.taskana</groupId>
         <artifactId>taskana-adapter-parent</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
+        <version>0.0.2-SNAPSHOT</version>
         <packaging>pom</packaging>
 
+        <name>${project.groupId}:${project.artifactId}</name>
```

Comment:
I would prefer that we use the same sequence of tags as we use them in taskana-parent. This would make comparing easier.

Reply:
what do you mean?